wordpress-mobile/gutenberg-mobile
# Correct color selection

Fixing the bug where color selection disappears when switching between solid/gradient segments.

ref to gutenberg: https://github.com/WordPress/gutenberg/pull/22736

To test:

1. Open mobile app
2. Add `Button` block
3. Open `Button` settings
4. Go to background color
5. Select any solid color
6. Press gradient segment
7. Press solid segment
8. Expect selection is visible (check mark on white background)

PR submission checklist:
